WordPress/src/main/java/org/wordpress/android/ui/posts/PostUtils.java

```diff
@@ -37,6 +39,8 @@
 
     private static final String GUTENBERG_BLOCK_START = "<!-- wp:";
 
+    public static final String TAG_GUTENBERG_CONFIRM_DIALOG = "tag_gutenberg_confirm_dialog";
```

Comment:
If we need the TAG, defining it here seems out of place. Can we place it in `GutenbergWarningFragmentDialog.kt` and just have `.initialize()` set it directly without providing it as a parameter?